src/Makefile

```diff
@@ -116,7 +116,7 @@ ifeq (${FC},nvfortran)
                 OPTS_M += -acc -gpu=cc80,lineinfo,nofma,math_uniform,managed -Mpcast -Minfo=accel
             else
                 OPTS += -acc -gpu=cc70,lineinfo,nofma,math_uniform -Mpcast -Minfo=accel
-                OPTS_M += -acc -gpu=cc70,lineinfo,nofma,math_uniform,managed -Mpcast -Minfo=accel
+                OPTS_M += -acc -gpu=cc70,lineinfo,nofma,math_uniform -Mpcast -Minfo=accel
```

Comment:
Do we want to maintain an OPTS_M compiler flag in the Makefile now that we do not need the use of Managed memory?

Reply:
Thanks @johnmauff. This is a good point. I keep it here in case we want to apply separate flags to different files in the future. But it does seem unnecessary by now since we no longer need the managed memory option. Shall we just remove it?

Reply:
The Makefile has been updated to remove the `OPTS_M` option.